src/components/auth/AuthInterceptor/index.tsx

```diff
@@ -20,8 +18,8 @@ interface IAuthInterceptorProps {
  *      2) 401에러가 뜨는 경우 refresh 요청
  */
 const AuthInterceptor = ({ children }: IAuthInterceptorProps) => {
-  const navigate = useNavigate();
-  const [accessToken, setAccessToken] = useRecoilState(accessTokenAtom);
+  const accessToken = useRecoilValue(accessTokenAtom);
+  const { mutateAsync } = useRefresh();
 
   const responseInterceptor = AuthAxios.interceptors.request.use((config) => {
```

Comment:
```suggestion
  const responseInterceptor = AuthAxios.interceptors.request.use((config) => {
  const requestInterceptor = AuthAxios.interceptors.request.use((config) => {
```
이 부분 변수명이랑 역할이 반대가 된 거 같아서 수정해봤습니다.
혹시 제가 잘못 이해한 부분이 있다면 말씀해 주시면 감사하겠습니다!

Reply:
변수명이 잘못된게 맞네요..ㅎㅎ 지금이라도 발견해서 다행입니다. 감사합니다!!